Survey/Sources/Presentation/Views/AnswerViews/AnswerViewModel.swift

```diff
@@ -37,20 +40,98 @@ extension AnswerViewModel: ViewModel {
             .assign(to: \.likelyLabelOpacity, on: output)
             .store(in: &output.cancelBag)
 
+        input.selectedAnswers
+            .compactMap { $0 }
+            .map { anwser in
+                let isMultipleChoices = ((displayType == .choice && pickType == .any) || displayType == .textfield)
+                handleSelectedAnswers(isMultipleChoices: isMultipleChoices, selectedAnswer: anwser)
+            }
+            .map { _ in true }
```

Comment:
Is this always set as true? If so could you check if we can define `didAnswer` as `void`?

Reply:
Thank you. I redefined `didAnswer` as `Void` at https://github.com/nkhanh44/ic-survey-ios/pull/85/commits/1942802d0f9be07a1d7eab503f21badf56c883ee.